javascript/javascript2/week3/warmUp.js

```diff
@@ -0,0 +1,100 @@
+// 1. Log out the text Called after 2.5 seconds 2.5 seconds after the script is loaded.
+function timeFunction() {
+  setTimeout(() => {
+    console.log("Called after 2.5 seconds");
+  }, 2500);
+}
+
+// 2. Create a function that takes 2 parameters: delay and stringToLog.
+// Calling this function should log out the stringToLog after delay seconds.
+// Call the function you have created with some different arguments.
+// const logStringAfterDelay = (delay, stringToLog) => {
+//   setTimeout(() => console.log(stringToLog), delay * 1000);
+// }
+function logStringAfterDelay(delay, stringToLog) {
+  setTimeout(() => console.log(stringToLog), delay * 1000);
+}
+logStringAfterDelay(3, "it!");
+logStringAfterDelay(1, "I");
+logStringAfterDelay(2, "did");
```

Comment:
To make it even more clear, you could use the parameter name `delayInSeconds` instead of `delay`, since you're converting the time from milliseconds to seconds. It's a good practice to being able to know how to use a function by just looking at the parameters and not having to look at the logic of the function itself.

But great job! Really good function name also.